<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnitDialog</class>
 <widget class="QDialog" name="UnitDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>102</width>
    <height>137</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <widget class="QGroupBox" name="infoBox">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="noLabel">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;黑体&quot;;</string>
          </property>
          <property name="text">
           <string>编号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="name">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;黑体&quot;;</string>
          </property>
          <property name="text">
           <string>名称</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="hpLabel">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;黑体&quot;;</string>
          </property>
          <property name="text">
           <string>血量</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="mpLabel">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;黑体&quot;;</string>
          </property>
          <property name="text">
           <string>移动力</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ceLabel">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;黑体&quot;;</string>
          </property>
          <property name="text">
           <string>攻击力</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections/>
</ui>
